homeassistant/components/ue_smart_radio/media_player.py

```diff
@@ -100,7 +100,6 @@ def __init__(self, session, player_id, player_name):
         self._session = session
         self._player_id = player_id
         self._name = player_name
-        self._state = None
         self._volume = 0
```

Comment:
The other properties are left in the old method, which is inconsistent. Let's ensure we keep it consistent in how integrations work.